components/client/Dockerfile

```diff
@@ -11,3 +11,7 @@ RUN pip install -e /app/fedn
 RUN chown -R fednuser /app
 USER fednuser
 WORKDIR /app
+
+# Clone the dashboard files into the docker container and install package drom dev branch
+RUN git clone --branch development https://github.com/flask-dashboard/Flask-MonitoringDashboard.git
```

Comment:
I think we should remember to refactor here, so that we have separate images for clients, combiners, the server.

Reply:
This is also for the new dashboard, development branch to let CPU loads work (CPU not supported in pypi package).